src/penn_chime/charts.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
```

Comment:
Remove the #!/usr/bin/python unless it is a script.

Reply:
done, will be in next push